certification/pyxis/submit.go

```diff
@@ -17,31 +20,42 @@ func (p *pyxisClient) SubmitResults(ctx context.Context, certInput *certificatio
 	certImage := certInput.CertImage
 
 	// Submission effectively starts the certification process, so switch
-	// the status to reflect this if needed.
+	// the status to reflect this if needed. This only needs to be done for net new projects.
+	// Existing projects that are in "In Progress" can stay "In Progress" until they moved to "Published" which is triggered
+	// once an image in a project is moved to "Published" status. The status on the project would stay in "Published" status,
+	// unless the partner decides to un-publish all of their images. At that point backed systems/processes would move
+	// the project back to "In Process" and there would still be nothing that preflight need to update on the project.
 	if certProject.CertificationStatus == "Started" {
```

Comment:
Yeah, I was wondering that as well, that's why I asked the CVP/Pyxis team. These are the scenarios they covered, so if there is another one that exists that they don't know about, that's kind of problematic, so hopefully they explained all of them and I didn't lose any in translation. I'm not sure if this would help, but maybe we need to have a meeting with them to explain all the backend systems and their role in this process.